modules/installation-creating-aws-worker.adoc

```diff
@@ -80,8 +86,7 @@ file metadata, which has the format `<cluster-name>-<random-string>`.
 <3> Current {op-system-first} AMI to use for the worker nodes.
 <4> Specify an `AWS::EC2::Image::Id` value.
 <5> A subnet, preferably private, to launch the worker nodes on.
-<6> Specify a subnet from the `PrivateSubnets` value from the output of the
-CloudFormation template for DNS and load balancing.
+<6> Specify a subnet for your VPC.
```

Comment:
I think the original description is more clear and reasonable. Why we need such change here?

Reply:
Good point. I've restored the original comment.